src/py21cmfast/src/ps.c

```diff
@@ -21,8 +21,12 @@ static gsl_spline *erfc_spline;
 
 #define NSFR_high 200
 #define NSFR_low 250
-#define NGL_SFR 100
-
+//#define NSFR_high 50 //200
+//#define NSFR_low 50 //250
```

Comment:
Remove debugging code and alternate values etc. Only include the fiducial values used for the calculations